### PR TITLE
Declare license as ZPL version 2.1

### LINKED ISSUE #15: Project's license: redeclare as PSF-2.0
## FEATURE REQUEST

_TL;DR: Python 2.1.1 is not a valid SPDX identifier, would it be possible to redeclare the project as PSF-2.0?_

Hello,
Fedora Linux is going through the process of declaring the licenses according to the SPDX standard. 
I was pointed here from docutils where we've came across the `roman`'s license: Python 2.1.1, which is not an existing SPDX identifier. Having asked Fedora Legal for an opinion, it was pointed out that the original declaration by Mark Pilgrim was this:

> "The example programs in this book are free software; you can redistribute and/or modify them under the terms of the Python license as published by the Python Software Foundation. A copy of the license is included in Appendix H, Python license."

And suggestion by the Legal team member:

> I would suggest they replace the Python 2.1.1 license text with some sort of instantiation of the PSF-2.0 license text, and possibly attempt to get in touch with Mark Pilgrim to confirm that this is appropriate.

Would it be possible?

[docutils issue](https://sourceforge.net/p/docutils/feature-requests/95/)
[Fedora Legal issue](https://gitlab.com/fedora/legal/fedora-license-data/-/issues/216#note_1387584572)

### PR DESCRIPTION
**Pull request by icemac**
Fixes #15.

### CODE REVIEW

**Review by icemac — COMMENTED** [review]
I like the changes, but I cannot approve the PR because I created it.

**Review by jugmac00 — APPROVED** [review]

**Comment by dataflake** [comment]
I'd like to wait for a comment from @befeleme before merging

**Review by befeleme — APPROVED** [review]
LGTM, thank you all.

**Pull request merged**
